aiohttp/web.py

```diff
@@ -318,10 +319,18 @@ def __repr__(self):
         return "<Application 0x{:x}>".format(id(self))
 
 
+class TerminateSignal(SystemExit):
+    code = 143
+
+
+def sigterm_handler(signum, frame):
+    raise TerminateSignal()
+
+
 def run_app(app, *, host=None, port=None, path=None, sock=None,
             shutdown_timeout=60.0, ssl_context=None,
             print=print, backlog=128, access_log_format=None,
-            access_log=access_logger, loop=None):
+            access_log=access_logger, handle_signals=True, loop=None):
```

Comment:
Yes actually I was thinking that you may want actually that run_app handles all the signals or none. For the user it's not really about handling SIGTERM in particular, it's more about handling the signals (INT and TERM) or not. It's unfortunate somehow that Python handles SIGINT for you by default.

I just realize now that maybe we should also handle SIGINT using https://docs.python.org/3/library/asyncio-eventloop.html#unix-signals but I'm not sure at all about it, what do you think?